environment.yml

```diff
@@ -13,5 +11,6 @@ dependencies:
 - numpy>=1.12.1
 - pandas>=0.20.1
 - pytest
-- python=2.7
```

Comment:
I think it would be nice to declare `python=3.6` as a dependency to encourage people to use it moving forward. Might make it easier to fully drop `2.7` when the time comes